components/UI/modal/index.jsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classes from 'components/UI/modal/modal.module.scss';
+import Backdrop from 'components/UI/backdrop';
+
+const Modal = ({ show, modalClosed, children }) => (
```

Comment:
Doesn't `modalClosed` sound weird because of past tense? How about `closeModal`?